backend/ibutsu_server/db/upgrades.py

```diff
@@ -102,7 +116,7 @@ def upgrade_3(session):
     if (
         "runs" in metadata.tables
         and artifacts is not None
-        and not hasattr(artifacts.columns, "run_id")
+        and artifacts.columns.contains_column("run_id")
```

Comment:
Why has this been changed?

Reply:
I changed this to `"run_id" in table.columns.keys()`